ansible_collections/arista/avd/roles/eos_designs/python_modules/network_services/router_internet_exit.py

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) 2023-2024 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+from __future__ import annotations
+
+from functools import cached_property
+
+from ansible_collections.arista.avd.plugins.plugin_utils.utils import get
+
+from .utils import UtilsMixin
+
+
+class RouterInternetExitMixin(UtilsMixin):
+    """
+    Mixin Class used to generate structured config for one key.
+    Class should only be used as Mixin to a AvdStructuredConfig class
+    """
+
+    @cached_property
+    def router_internet_exit(self) -> dict | None:
+        """
+        Return structured config for router_internet_exit
+
+        Only used for CV Pathfinder routers today
+        """
+        if not self.shared_utils.is_cv_pathfinder_router:
+            return None
+
+        router_internet_exit = {}
+        exit_groups_dict = {}
+        policies = []
+
+        for policy in self._filtered_internet_exit_policies:
+            policy_exit_groups = []
+            for connection in policy.get("connections", []):
```

Comment:
Add TODO as right now we rely on the order in connections. Need to make sure the order inside the policy is the expected one